### PR TITLE
14/01/2021, show all product admin

### PR DESCRIPTION
**Pull request by haupt-2855**
## Related Tickets
- [#45003](https://edu-redmine.sun-asterisk.vn/issues/45003)

## WHAT (optional)
- Create layout  all products in admin page
- Show all product in admin page.
- 
## Evidence (Scr
Screenshot from 2022-01-15 13-28-47
eenshot or Video)
Screenshot from 2022-01-15 13-29-04

### CODE REVIEW

**Comment by haupt-2855** [comment]
ready

File: app/controllers/admin/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Admin::ProductsController < Admin::BaseController
+  def index
+    @pagy, @products = pagy Product, items: Settings.per_page_10
```
Comment (vanvtt-0952) [inline app/controllers/admin/products_controller.rb]: Màn hình list thì nên có sự sắp xếp nhé

File: app/views/admin/products/_all_products.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<% @products.each_with_index do |product, i| %>
+  <tr>
+    <td scope="row">
+      <%= i + 1 + (params[:page].to_i - 1)*Settings.per_page_10 %>
```
Comment (vanvtt-0952) [inline app/views/admin/products/_all_products.html.erb]: viết method trong helper đi cụ

File: app/views/admin/products/_all_products.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    </td>
+    <td><%= product.name %></td>
+    <td><%= product.price %></td>
```
Comment (vanvtt-0952) [inline app/views/admin/products/_all_products.html.erb]: format tiền tệ nào

**Comment by haupt-2855** [comment]
ready

**Pull request merged**